tests/network/vmi_multus.go

```diff
@@ -72,7 +72,7 @@ const (
 const (
 	sriovnet1           = "sriov"
 	sriovnet2           = "sriov2"
-	sriovnetLinkEnabled = "sriov-link-enabled"
+	sriovnetLinkEnabled = "sriov-linked"
```

Comment:
This will end up being the network name, spec interface name, and multus network name (the name of the NAD).

I don't understand (I'm ignorant when it comes to sriov) how this ends up being the interface name in the guest.

Reply:
And no error on cloud-init output